Annotato/Annotato/ViewModels/Auth/AuthViewModel.swift

```diff
@@ -34,23 +43,19 @@ class AuthViewModel {
         }
     }
 
-    func logIn(email: String, password: String) {
-        authService.logIn(email: email, password: password)
-    }
-
-    private func setUser(userId: String = "") {
+    private func handleLogin(userId: String) {
```

Comment:
What's the difference between `logIn` and `handleLogin`?

Reply:
LogIn is to call the authservice login. Handle login is to handle a successful login. Renamed to `handleLoginSuccess`.